src/App.css

```diff
@@ -1,41 +1,40 @@
-.App {
-  text-align: center;
-  max-width: 100%;
+html {
+  height: 100%; 
+}
+
+body {
+  height: 100%; 
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.root {
+  height: 100%; 
 }
```

Comment:
Might have to convert these to rem so the font sizes aren't affected but we can go over that momentarily